molo/core/admin.py

```diff
@@ -67,9 +85,207 @@ def articles(self, obj, *args, **kwargs):
     def get_queryset(self, request):
         qs = ArticlePage.objects.descendant_of(
             request.site.root_page).filter(
-                languages__language__is_main_language=True).exclude(
-                    reaction_questions=None)
```

Comment:
Ah wow thanks for spotting that I removed it to test and forgot to put it back, fixing...